test/Base_Tests/src/Semantic/Multi_Value_Spec.enso

```diff
@@ -14,6 +35,61 @@ add_specs suite_builder =
             (c:Complex).re . should_equal 1.5
             (c:Float) . should_equal 1.5
 
+    suite_builder.group "Chain Multi Value" group_builder->
+        to_b_to_c obj =
+            b = obj : B
+            c = b : C
+            c
+
+        group_builder.specify "Multiple conversions of A work" <|
+            a = A
+            a_text = to_b_to_c a
+            a_text . to_text . should_equal "(C (B A))"
+
+        group_builder.specify "Multiple conversions of A & X work" <|
+            ax = A:A&X
+            a_text = to_b_to_c ax
+            a_text . to_text . should_equal "(C (B A & X))"
```

Comment:
I'm trying to understand where the `A & X` is coming from in this text representation and I really cannot find it. Can I ask for any pointers?

Reply:
Ahhh I see, this is `EnsoMultiValue::toString` (called from its `toDisplayString`).

Hmm, I'm not quite sure that this behaviour is correct. Does this affect only internal `toDisplayString` or does it affect `to_text` as well?

I'd like to add a test:
```
ax = A:A&X
ax.to_text . should_equal "A"
```

One of the ideas for multi value is to be able to do `c : Column & Integer_Column`. In that case, I think we want `to_text` to be dispatched 'normally' and not include any `&` intersections.

Reply:
This is probably outside of the scope of this PR, but yeah, the `multi_value.to_text` needs to be meaningfully defined. But what it should be?
- names of all intersection types - as now - ignores `.to_text` of all the values
- `.to_text` of the first value - looses information about the others
- concatenation of all `.to_text` values - may get messy

Reply:
IMO `to_text` of first type is best. It does hide the 'other values' and that is not ideal, but given the kind of use cases we want to use multi-values for (e.g. the `Column & Integer_Column`), I think that delegating to the first type is the best we can do to keep this sensible.

In that regard, perhaps `to_text` shouldn't be any different than e.g. `to_js_object` or `to_defautl_visualization_data` that we use to visualize the values. These will also delegate to the first value.

Reply:
- reported as #11827